dali/operators/image/paste/paste.h

```diff
@@ -64,11 +66,60 @@ class Paste : public StatelessOperator<Backend> {
     return false;
   }
 
-  void RunImpl(legacy_workspace_t<Backend> &ws) override;
+  void SetupSampleParams(Workspace &ws) {
```

Comment:
`test_paste_cpu` fails without a GPU. L45 sets stream to `fill_value_`  which is also pinned by default. For CPU op I would set it non pinned, for GPU we can run L45. Maybe there are other things, but this struck me right away.

Reply:
Thanks, let me check that in the CI.

Reply:
It works, thank you!